theatre/studio/src/store/types/historic.ts

```diff
@@ -71,6 +71,9 @@ export type StudioHistoricState = {
           {
             selectedInstanceId: undefined | string
             sequenceEditor: {
+              markers?: {
+                time: number
+              }[]
```

Comment:
Change: It seems like markers should individually have their own identifiers as well (something as simple as `Date.now().toString(36)` would be fine to generate an id in mvp). This would ensure we can talk about markers based on their `id` rather than just the time they sit on (preventing multiple markers from being in the same place).

Reply:
Side note: This could be fun to use `Nominal`s on after my other refactor eventually lands.

Reply:
Yeah some identifier would be necessary to track concurrent changes between multiple users. Have a look at `generateKeyframeId()` for an example.

Reply:
Yeah some identifier would be necessary to track concurrent changes between multiple users. Have a look at `generateKeyframeId()` for an example.

Reply:
I'd say just add an identifier like `{time: number, id: string}`. No need to do `{list: ID[], byId: Record<ID, {time: number}>}`. The latter is more efficient, but there won't be that many markers per sequence anyway.